django-stubs/contrib/auth/hashers.pyi

```diff
@@ -15,30 +15,55 @@ def identify_hasher(encoded: str) -> BasePasswordHasher: ...
 def mask_hash(hash: str, show: int = ..., char: str = ...) -> str: ...
 
 class BasePasswordHasher:
-    algorithm: str
-    library: str | tuple[str, str]
+    algorithm: None
+    library: None
```

Comment:
```suggestion
    library: library: str | tuple[str, str] | None
```

Reply:
I applied this changes, thanks for your suggestion